src/coreclr/jit/gentree.cpp

```diff
@@ -11434,6 +11434,9 @@ void Compiler::gtDispConst(GenTree* tree)
                         case GTF_ICON_STATIC_BOX_PTR:
                             printf(" static box ptr");
                             break;
+                        case GTF_ICON_STATIC_ADDR_PTR:
+                            printf(" static fld addr");
```

Comment:
```suggestion
                            printf(" static base addr cell");
```
Otherwise it sounds like "address of the static field", which it definitively is not.